zipline/test_algorithms.py

```diff
@@ -316,8 +318,8 @@ def handle_data(self, data):
                             data[0].price,
                             style=StopLimitOrder(10, 10))
 
-            assert len(self.blotter.open_orders[0]) == 1
-            result = self.blotter.open_orders[0][0]
+            assert len(self.blotter.open_orders[self.sid(133)]) == 1
```

Comment:
Is the reason to switch to sid(133) because that is the sid for which the data_portal helper is creating trades?